dev/namrathagopalabhatla/visualize_eval_metric.py

```diff
@@ -0,0 +1,74 @@
+import numpy as np
+import pandas as pd
+
+import matplotlib.pyplot as plt
+
+from data_processing import split_data
+from classification_models import Classifier
+
+
+def generate_data(n=15, dist="gaussian", scale=0.5, prob=0.2):
+    """Function to generate data of different types of distributions"""
+    if n < 3:
+        print("Too few points, minimum n required is n > 2.")
+        return None
+
+    X = np.arange(15)
+    data = []
+
+    for i in range(len(X)):
```

Comment:
This could just be `range(15)`, and then `i` instead of `X[i]` on line 21. Although, really the `15` (number of rows) should be passed as a function arg.